### ISSUE TITLE
Previous work on DwC-MIxS extension

### ISSUE THREAD

**Issue by raissameyer** [tdwg/gbwg#10]
Hi @timrobertson100 and @thomasstjerne,

Looking at the MIxS-relevant part of this [collection of extensions](https://tools.gbif.org/dwca-validator/extension.do?id=http://gensc.org/ns/mixs/terms/Sample) that GBIF offers, I noted that they vary in scope and that some focus on MIxS, while others also cover parts of GGBN and MIQE, however I'm wondering how they relate to each other. 

Are these extensions sequential or are they independent? 

And how do they, including GBIF’s DNA derived data extension relate to the 2012 extension built by @tucotuco et al.?

Best, 
Raïssa

**Comment by gdadade** [comment]
All extensions starting with "GGBN" were developed by GGBN. Since our data standard already includes MIxS terms, those are also included in those extensions. They can all be used independently. Re the others I can't give advice, as I wasn't involved in the development.

Best,
Gabi

**Comment by thomasstjerne** [comment]
Hi Raïssa
If you take a look at the GBIF sandbox:
https://rs.gbif.org/sandbox/extension/
You will see a series of mixs_sample extensions. The one that has no date stamp: https://rs.gbif.org/sandbox/extension/mixs_sample.xml is the 2012 extension built by @tucotuco et al.

The [2019 versions](https://rs.gbif.org/sandbox/extension/mixs_sample_2019_10_04.xml) represent a rather clean update to MIxS 4 of the original.

The [2020 versions](https://rs.gbif.org/sandbox/extension/mixs_sample_2020_06_23.xml) represent an update to MIxS 5, and additions of fields such as `pcr_primer_reference`, several MIQE related fields and some fields from GGBN.

In october 2020 the extension was renamed from `mixs_sample` to `dna_derived_data`.

In summary, the [DNA derived data extension](https://rs.gbif.org/sandbox/extension/dna_derived_data.xml) represent the latest version of this work and substitutes all the date stamped `mixs_sample` versions. You will find the associated vocabularies for this extension here: https://rs.gbif.org/sandbox/vocabulary/mixs/

Best,
Thomas

**Comment by timrobertson100** [comment]
Following today's call @thomasstjerne will update the latest version which is the [DNA derived data extension](https://rs.gbif.org/sandbox/extension/dna_derived_data.xml) with the correct IRIs for MIxS and then we'll move it to this GitHub repository for further work and remove all the historical versions from the sandbox to avoid confusion, referring to the version here as the authoritative one.
